modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.tsx

```diff
@@ -53,6 +53,12 @@ const Cell = ({
 		if (columnName && heading && !isFixed && cellRef.current) {
 			const boundingClientRect = cellRef.current.getBoundingClientRect();
 
+			if (columnName !== 'item-actions') {
+				if (boundingClientRect.width < 100) {
+					boundingClientRect.width = 140;
+				}
+			}
```

Comment:
We can simplify this, and it's kinda odd to have this depression of different size between 100 and 140. I'm thinking something like 
```js
if (columnName !== 'item-actions' && boundingClientRect.width < 140) {
		boundingClientRect.width = 140;
}
```

Reply:
Made the changes. 
Thank you!